odoo/addons/base/models/ir_ui_view.py

```diff
@@ -347,23 +361,24 @@ def _compute_invalid_locators(self):
                 if spec.tag == 'data':
                     specs.extend(spec)
                     continue
-                # Capture 'move' nodes to handles cases where move operations are nested within other xpath operations
-                # <xpath expr="parent" position="after">
-                #   <xpath expr="child" position="move"/>
-                # </xpath>
-                specs.extend(c for c in spec if c.get("position") == 'move')
-                node = None
-                with suppress(ValidationError):  # Syntax error
-                    # If locate_node returns None here:
-                    # Invalid expression: Ok Syntax, but cannot be anchored to the parent view.
-                    node = self.locate_node(source, spec)
-                if node is None:
-                    invalid_locators.append({
-                        "tag": spec.tag,
-                        "attrib": dict(spec.attrib),
-                        "sourceline": spec.sourceline
-                    })
+
+                invalid_locator = assess_locator(source, spec)
+                if invalid_locator is not None:
+                    invalid_locators.append(invalid_locator)
```

Comment:
nitpicking (walrus + more consistent condition with the one below)
```suggestion
                if invalid_locator := assess_locator(source, spec):
                    invalid_locators.append(invalid_locator)
```